pb/protos/events.proto

```diff
@@ -902,6 +903,11 @@ message Event {
                 string id = 1; // dataview block's id
                 bool value = 2;
             }
+
+            message ActiveViewIsSet {
```

Comment:
better be ActiveViewSet? because the last in our naming is just a verb

Reply:
Thank you, done!